New 2018 TIGER data and moved conversion scripts

mtmail opened the pull request:
Downloading the TIGER data from their FTP server took me 12h, converting another several hours. It's unlikely the average server admin will choose the extra work when a preprocessed file is mentioned in the install instructions.

This PR documents how the preprocessed file got created in a new `data-sources/us-tiger/` directory and also moves the related scripts there. I'm open for suggestions on naming the directories and files.

`utils/imports.php` was only a small wrapper around the TIGER conversion script and is no longer needed. One can argue it also had the wrong name as it didn't import anything (that logic is in `utils/setup.php --import-tiger-data`).

You can fetch an updated preprocessed file on https://downloads.opencagedata.com/public/

I imported the 46* files in the nominatim api test database and updated one test case.

lonvia commented on docs/admin/Import-and-Update.md:
````diff
@@ -101,52 +101,34 @@ Note that this command downloads the phrases from the wiki link above.
     ./utils/setup.php --create-functions --enable-diff-updates --create-partition-functions
 ```
 
-The entire US adds about 10GB to your database.
````
Don't remove this sentence.

mtmail replied:
Wasn't removed, just moved a paragraph higher up.

lonvia commented on data-sources/us-tiger/convert.sh:
```diff
@@ -0,0 +1,50 @@
+mkdir -p "$OUTPATH"
+
+WORKPATH="/tmp/tiger2018work"
+mkdir -p "$WORKPATH"
```
I'd like to avoid hardcoded paths. How about changing the script to take two mandatory parameters 'inpath' and 'outpath' and then create a temporary work path under outpath.

lonvia commented on data-sources/us-tiger/convert.sh:
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set +x
```
Debug option that should be removed.

mtmail replied:
removed

lonvia commented on data-sources/us-tiger/convert.sh:
```diff
@@ -0,0 +1,50 @@
+if [[ ${#INFILES[*]} != $NUM_EXPECTED ]]; then
+    echo "Expected $NUM_EXPECTED files. Found ${#INFILES[*]}."
+    exit 1
+fi
```
I don't think that's a good idea because a) there is no guarantee that the number does not change over the years and then the script becomes incompatible and b) sometimes one only wants to convert a couple of files for testing.

lonvia commented:
While you are moving stuff around for Tiger, do you mind looking into #1199? I would suggest to add a new utility for any imports and updates of external data.

mtmail commented:
Added your feedback to the script. Moved (symlink) documentation into /docs/data-sources I think deserves its own section since we don't expect admins to run this scripts often if at all.

lonvia commented:
Merged manually.
